packages/mdc-snackbar/index.js

```diff
@@ -47,6 +47,16 @@ export class MDCSnackbar extends MDCComponent {
       unsetActionAriaHidden: () => getActionButton().removeAttribute('aria-hidden'),
       setActionText: (text) => { getActionButton().textContent = text; },
       setMessageText: (text) => { getText().textContent = text; },
+      setFocus: () => getActionButton().focus(),
+      visibilityIsHidden: () => document.hidden,
+      registerBlurHandler: (handler) => getActionButton().addEventListener('blur', handler, true),
```

Comment:
Again, I don't think so. What matters is that we're reacting to a `blur` event. I'll document that it happens during the capture phase